indico/modules/users/controllers.py

```diff
@@ -426,6 +427,27 @@ def _process(self):
         return jsonify(self.user.settings.get('use_markdown_for_minutes'))
 
 
+class RHUserPreferencesMastodonServer(RHUserBase):
+    def _validate_server_url(self):
```

Comment:
I don't think this should be called `self`, it's not a method on the class after all.
Maybe best to move it outside the class...

Reply:
I'm considering to also tweak the validation a bit in both here and the user preferences section: I found that users can still pass in URLs with paths (e.g. `https://mastodon.social/a/b/c/`) that breaks the sharing on Mastodon's side.